### ISSUE TITLE
Automatic e-mail signatures for all active members

### ISSUE THREAD

**Issue by jeffreybakker** [Inter-Actief/amelie#589]
Beheer got a request whether it was possible to automatically install e-mail signatures for all active members so that the e-mail signatures are uniform and for "uitstraling".

**Example**
![Screenshot 2021-11-12 at 15 38 29](https://user-images.githubusercontent.com/5832333/141484352-e59ccc0f-a826-4054-987c-ca740ddec870.png)

**Comment by WKobes** [comment]
I do not think this is an amelie issue as this should probably be configured in Google Workspaces directly.

Maybe relevant: https://support.google.com/a/answer/176652

**Comment by Kurocon** [comment]
Needs to be set using the Gmail API according to that page, probably per individual users. So this will probably need to be done when a user is created via Claudia.

It also notes that Google Workspace is required, and we might still be on the old GSuite, so some investigation is needed to see if we need to upgrade, and if so, if we keep our 'unlimited' free plan or not.

**Comment by Kurocon** [comment]
Should be doable with our current Google Workspaces, and should be pretty easy.

**Comment by MiniLars** [comment]
@bartoostveen if you're working on this, can you make one that is a bit less ugly? 

We as Board (and several other committees) have this, you could use it as inspiration:

![Screenshot_20260626_141038_Gmail.jpg](https://github.com/user-attachments/assets/9159a931-adca-465d-b91a-30e13ff9af1d)

**Comment by WKobes** [comment]
> [@bartoostveen](https://github.com/bartoostveen) if you're working on this, can you make one that is a bit less ugly?
> 
> We as Board (and several other committees) have this, you could use it as inspiration:
> 
> ![Screenshot_20260626_141038_Gmail.jpg](https://github.com/user-attachments/assets/9159a931-adca-465d-b91a-30e13ff9af1d)

Hi @ItzLarz, would advise against using images in e-mail signatures. They cause a vast increase in e-mail size (think of a factor 10 at least for every e-mail sent) which is basically a waste of bandwith / energy. Several e-mail clients will mess up the formatting as well, especially in table form. Best practice for e-mail signatures is basically KISS.
